src/test/java/org/imsglobal/caliper/partner/events/RenLearnStarScoreEventGradedTest.java

```diff
@@ -0,0 +1,160 @@
+ * Software Foundation, version 3 of the License.
+ *
+ * IMS Caliper is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.imsglobal.caliper.partner.events;
+
+import static com.yammer.dropwizard.testing.JsonHelpers.jsonFixture;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.ACTIVITY_REF_ID;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.APP_REN_LEAN_NAME;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.BASE_IRI;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.BASE_URN;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.DISTRICT_REF_ID;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.OBJECT_ID;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.SCHOOL_REF_ID;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.STUDENT_USER_REF_ID;
+import static org.imsglobal.caliper.partner.events.RenLearnConstants.LAST_ATTEMPT_ID;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.imsglobal.caliper.TestUtils;
+import org.imsglobal.caliper.actions.Action;
+import org.imsglobal.caliper.context.JsonldContext;
+import org.imsglobal.caliper.context.JsonldStringContext;
+import org.imsglobal.caliper.entities.EntityType;
+import org.imsglobal.caliper.entities.agent.Membership;
+import org.imsglobal.caliper.entities.agent.Organization;
+import org.imsglobal.caliper.entities.agent.Person;
+import org.imsglobal.caliper.entities.agent.Role;
+import org.imsglobal.caliper.entities.agent.SoftwareApplication;
+import org.imsglobal.caliper.entities.agent.Status;
+import org.imsglobal.caliper.entities.outcome.Score;
+import org.imsglobal.caliper.entities.resource.Assessment;
+import org.imsglobal.caliper.entities.resource.Attempt;
+import org.imsglobal.caliper.events.GradeEvent;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+@Category(org.imsglobal.caliper.UnitTest.class)
+public class RenLearnStarScoreEventGradedTest {
+    private JsonldContext context;
+    private String id;
+    private String member_id;
+    private String generated_id;
+    private String generated_name;
+    private SoftwareApplication edApp;
+    private Person actor;
+    private Attempt object;
+    private Assessment assignable;
+    private Score generated;
```

Comment:
`Score` is a type of `Generated` entity, so it's probably clearer to have it's variable name as `score` rather than `generated` as it describes it clearly.

Reply:
Fixed